src/styles/global.scss

```diff
@@ -1,17 +1,46 @@
 @use './typography';
 @use './variables' as *;
+@use './mixins';
 @use './transitions';
 
+* {
+  box-sizing: border-box;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+}
+
+html,
+body {
+  height: var(--height);
+}
+
+body {
+  margin: 0;
+  background-color: $color-black;
+
+  &.color-bg-3 {
+    background-color: $color-bg-3;
+  }
+
+  @include mixins.desktop {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```

Comment:
You could fit it in one line by using flex mixin

Reply:
I know that having less code usually is a better idea. But SCSS mixins after all are still transpiled into regular CSS rules. So there's no benefit in speed. And in my opinion makes the code harder to read.